packages/react/runtime/src/auto.ts

```diff
@@ -146,11 +146,15 @@ const dispatcherMachinePROD = createMachine({
 ```
 */
 
+export let isAutoSignalTrackingInstalled = false;
+
 let store: EffectStore | null = null;
 let lock = false;
 let currentDispatcher: ReactDispatcher | null = null;
 
 function installCurrentDispatcherHook() {
```

Comment:
this function still called on top level of `@preact/signals-react` so i there is a difference

Reply:
This PR doesn't change the functionality of `@preact/signals-react` in that importing that package still hooks into internals. Changing that behavior will come later since it is a break change. This PR is starting to setup some infra to eventually make that behavior (hooking into React internals) opt-in. So no difference at the moment for people using `@preact/signals-react` but that will change eventually.